tmc-langs-util/src/main/java/fi/helsinki/cs/tmc/langs/util/TaskExecutor.java

```diff
@@ -0,0 +1,22 @@
+package fi.helsinki.cs.tmc.langs.util;
+
+import com.google.common.base.Optional;
+import fi.helsinki.cs.tmc.langs.ExerciseDesc;
+import fi.helsinki.cs.tmc.langs.RunResult;
+import fi.helsinki.cs.tmc.stylerunner.validation.ValidationResult;
+import java.nio.file.Path;
+
+
+public interface TaskExecutor {
```

Comment:
Please add a comment about what this is and why.

I think it's a good idea to separate command implementation from the argument handling like this.